### PR TITLE
Added support fot omitting code of module system

### PR DESCRIPTION
**Pull request by iniel**
There's another open PR on this issue (https://github.com/enb-bem/enb-bem-specs/pull/45), this one brings it up-to-date with the latest module changes.

### CODE REVIEW

File: lib/node-configurator.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ exports.configure = function (config, options) {
                 target: '?.pure.ym.js',
                 sourceSuffixes: [],
                 filesTarget: '?.base.files',
-                includeYM: true
+                includeYM: !options.omitModules
```
Comment (blond) [inline lib/node-configurator.js]: Can you, please, rename this option to `includeYM` with default value `true`.

**Comment by blond** [comment]
@iniel, ping?

**Comment by iniel** [comment]
@blond, updated.

**Comment by blond** [comment]
@iniel need rebase onto `master` branch.

**Comment by iniel** [comment]
@blond, rebased.

**Comment by blond** [comment]
:+1:

**Pull request merged**